### PR TITLE
Add translations POT file

### PR DESCRIPTION
**Pull request by cea2aj**
Add the POT file to the repo so that we can easily access it

Also update the translation verification script to throw an error if the extracted translations are out of date

NOTE: The translation test is failing because we don't have all the translations in yet.

J=none
TEST=manual

See that when the messages.pot file in the repo is out of date, the test fails

### CODE REVIEW

File: .circleci/run_translation_verification.sh [hunk 1]
```diff
@@ -1,9 +1,22 @@
 #!/bin/bash
 
+# Verifiy that the messages.pot file in the repo is up to date
+# If there are any git diffs after translations are extracted, the checked-in file is out of date
+
 npm run extract-translations
 
+git diff --exit-code > /dev/null # send stdout to /dev/null to reduce clutter in the CI output
```
Comment (oshi97) [inline .circleci/run_translation_verification.sh]: can we diff just messages.pot?

Comment (cea2aj) [reply .circleci/run_translation_verification.sh]: sure

**Review by oshi97 — APPROVED** [review]
~looks great!~ still looks great but the translation test and unsafe-eval snapshot are derping out

**Comment by cea2aj** [comment]
> ~looks great!~ still looks great but the translation test and unsafe-eval snapshot are derping out

the translation test is supposed to be failing because we don't have all the translations in yet

It looks like the unsafe-eval snapshot has been blank since that PR was merged in. (I didn't notice it because it was auto-approved by percy)

**Review by oshi97 — APPROVED** [review]

**Pull request merged**